content/en/blog/_posts/2023-03-08-Updates-to-the-Auto-refreshing-Official-CVE-Feed/index.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: blog
+title: Updates to the Auto-refreshing Official CVE Feed
+date: 2023-03-17
```

Comment:
```suggestion
date: 2023-03-22
```

There's a big [change](https://kubernetes.io/blog/2023/03/10/image-registry-redirect/) on Monday. Let's drop this one in after that's happened.